apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryClient.JobCrud.cs

```diff
@@ -238,6 +238,39 @@ public virtual BigQueryJob CreateExtractJob(TableReference tableReference, IEnum
         }
 
         /// <summary>
+        /// Creates a job to copy one table to another.
+        /// </summary>
+        /// <remarks>
+        /// To avoid confusion between source and destination tables, overloads are not provided that take the
+        /// individual components of table references. Instead, use <see cref="GetTableReference(string, string)"/> or
+        /// <see cref="GetTableReference(string, string, string)"/> to create table references.
+        /// </remarks>
+        /// <param name="source">The source table to copy. Must not be null.</param>
+        /// <param name="destination">The destination to copy to. Must not be null.</param>
+        /// <param name="options">The options for the operation. May be null, in which case defaults will be supplied.</param>
+        /// <returns>The job created for the copy operation.</returns>
+        public virtual BigQueryJob CreateCopyJob(TableReference source, TableReference destination, CreateCopyJobOptions options = null)
+            => CreateCopyJob(new[] { GaxPreconditions.CheckNotNull(source, nameof(source)) }, destination, options);
+
+        /// <summary>
+        /// Creates a job to copy data from at least one table to another.
+        /// This method creates a single-element array and delegates to <see cref="CreateCopyJob(IEnumerable{TableReference}, TableReference, CreateCopyJobOptions)"/>.
```

Comment:
Aargh... I deliberately removed it from one, but apparently the wrong one. I'm rubbish.